org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -640,38 +640,36 @@ private float IntersectQuad(Vector3 start, Vector3 end, float depth)
         }
 
         /// <summary>
-        /// Registers the duple Collider + XRBaseInteractor as triggering proximity.
+        /// Registers the interactable triggering proximity.
```

Comment:
Detector instead of interactable.  But preferable to just do `/// <inheritdoc/>`

Reply:
Updated in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/659/commits/b12b1cea8efa1f7b52f98b647c5bf697efe4e844, resolving.